app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field form-group col-md-6 my-2">
+    <%= f.label :name, class: 'form-label'  %><br />
+    <%= f.text_field :name, class: 'form-control' %>
+  </div>
+
+  <div class="field form-group col-md-6 my-2">
+    <%= f.label :email, class: 'form-label'  %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'form-control' %>
+  </div>
+
+  <div class="field form-group col-md-6 my-2">
+    <%= f.label :password, class: 'form-label'  %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class: 'form-control' %>
+  </div>
+
+  <div class="field form-group col-md-6 my-2">
+    <%= f.label :password_confirmation, class: 'form-label'  %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'form-control' %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up", class:'btn btn-info text-white' %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ]  @Team Great work adding the Signup page, however the design slightly differs from the design given, the buttons should be the same width as the filed form, check the comparison below:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/katarighe/blog-app/assets/61408860/e99fbec7-d31c-41b2-bd32-e6e01f9284bd) |  ![image](https://github.com/Nelofarzabi/blog_app/assets/61408860/4f4b8542-92b2-46f1-9b59-fc8c0f43c49f)